pages/levels/5.vue

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+
+const code = `    `;
+
+export default defineComponent({
+  data: () => {
+    return {
+      markdown: `What’s the deal with POSH? POSH stands for \'Plain Old Semantix HTML\'. It basically means you should use the right HTML elements for their intended purpose, like using a button instead of a div with a click function.
+
+But why is it so important to use the right element?
+Firstly, it makes it easier for people to navigate with a keyboard or screen reader if the technology can recognise what it’s navigating to. Secondly, it’s also smaller in file size and easier to make responsive when developing for mobile phones. Lastly, why would you make an effort to make everything accessible, but then not use the built-in accessibility from plain HTML elements?
+## Links
+- [HTML and accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML)`,
+
+      code,
+      defaultCode: code,
+      task: "Help - I can’t navigate!",
```

Comment:
Der falsche Strich (siehe jerabek)
```suggestion
      task: "Help – I can’t navigate!",
```